pkg/throttled_logger/throttled_logger.go

```diff
@@ -0,0 +1,39 @@
+package throttled_logger
```

Comment:
Don't use an underscore for package name, thats an anti-pattern, see https://go.dev/doc/effective_go#package-names